_config.yml

```diff
@@ -1,96 +1,25 @@
-# Site settings
-title: "The Union of UK Unicyclists"
-email: webmaster@unicycle.org.uk
-description: > # this means to ignore newlines until "baseurl:"
-  The website of the Union of UK Unicyclists
-baseurl: "" # the subpath of your site, e.g. /blog/
-url: "http://www.unicycle.org.uk" # the base hostname & protocol for your site
-twitter_username: UUUnicyclists
-github_username:  unionofukunicyclists
-facebook_page: theunionofukunicyclists
-google_maps_api_key: "AIzaSyBoPKA9DXHMbdvaqMPWxXGX1rUbAJN5JsI"
+# site settings
+title: The Union of UK Unicyclists
+shorttitle: UUU
+subtitle: The Union of UK Unicyclists
+email: committee@unicycle.org.uk
+description: The Union of UK Unicyclists promotes the sport and art of unicycling in the UK. 
+baseurl: /unionofukunicyclists.github.io # the subpath of your site, e.g. /blog
+#url: www.unicycle.org.uk # the base hostname & protocol for your site
+#url: mchonofsky.github.io # /unionofukunicyclists.github.io I think site.url is deprecated
```

Comment:
@mchonofsky 
I didn't see any evidence of this deprecation. There is a discussion at https://github.com/jekyll/jekyll/issues/4711 but it doesn't look like they reached a conclusion. `Also site.url` is still mentioned in Jekyll's documentation ( https://jekyllrb.com/docs/variables/#site-variables ). It probably doesn't matter what `site.url` is right now because I don't think the site actually uses it right now.

The comment you've left here suggests you're testing by pushing to your own local repo. I don't really recommend doing that. You can just run locally on your machine to test out changes.

Reply:
Also see https://jekyllrb.com/docs/upgrading/0-to-2/#custom-config-file